packages/components/pie-assistive-text/README.md

```diff
@@ -1,47 +1,98 @@
-3. [Documentation](#documentation)
-4. [Questions](#questions)
-5. [Contributing](#contributing)
+## Table of Contents
 
-## pie-assistive-text
+- [Installation](#installation)
+- [Documentation](#documentation)
+  - [Properties](#properties)
+  - [Slots](#slots)
+  - [CSS Variables](#css-variables)
+  - [Events](#events)
+- [Usage Examples](#usage-examples)
+- [Questions and Support](#questions-and-support)
+- [Contributing](#contributing)
 
-`pie-assistive-text` is a Web Component built using the Lit library.
+## Installation
 
-This component can be easily integrated into various frontend frameworks and customized through a set of properties.
+> To install any of our web components in your application, we would suggest following the [getting started guide](https://webc.pie.design/?path=/docs/introduction-getting-started--docs) to set up your project.
 
+Ideally, you should install the component using the **`@justeattakeaway/pie-webc`** package, which includes all of the components. Or you can install the individual component package.
 
-## Installation
+## Documentation
+
+### Properties
+
+| Prop              | Options                           | Description                                                                                                                                                                                                                                   | Default   |
+|-------------------|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| `variant`         | `"default"`, `"error"`, `"success"` | Sets the type of message displayed to one of three potential options: `error`, `success` or `default`. `error` and `success` include an icon and are used for validation messages. `default` provides users with extra context and guidance. | `default` |
+| `isVisuallyHidden`| `true`, `false`                    | If true, hides the component visually but leaves it accessible for screen readers.                                                                                                                                                           | `false`   |
+
+
+### Slots
+
+| Slot     | Description                                                  |
+|----------|--------------------------------------------------------------|
+| `default`| The default, unnamed slot is used to pass in text to the component. |
+
+### CSS Variables
+This component does not expose any CSS variables for style overrides.
+
+### Events
 
-To install `pie-assistive-text` in your application, run the following on your command line:
+This component does not emit any custom events. In order to add event listening to this component, you can treat it like a native HTML element in your application.
```

Comment:
nit; we don't expect components like assistive text and dividers to have any events. Do you think the section is misleading/confusing in this case?

Reply:
Maybe we can update the copy?
_This component does not emit any events_